sdk/include/opentelemetry/sdk/trace/always_off_sampler.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
```

Comment:
Let's sync with #122 to determine a proper location for those samplers. #122 puts the sampler implementation in `sampler.h`.

I prefer the interface definition to be in a separate file and not mixed with implementation, so the solution in this PR is preferable. Maybe we should have a `sdk/trace/samplers/always_off.h`?

Reply:
I like the idea of having the samplers in their own directory like you suggested. Let's touch base with @ziqizh during our sync and get his thoughts.

Reply:
Agreed. I thought we wanted to put everything in `sampler.h`, but making a separate folder looks good to me. I will move the AlwaysOnSampler to `sdk/trace/samplers/always_on.h`